gitChecks/constants.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+  API:'https://api.github.com/graphql',
```

Comment:
Github graphQL api has only one endpoint, I believe there is no problem in leaving this as a constant because it will always ask for a Oauth token so if a user doesn't have permissions to see a repository it will be validated with the token